lib/slots.ts

```diff
@@ -12,17 +12,69 @@ const getMinutesFromMidnight = (date) => {
   return date.hour() * 60 + date.minute();
 };
 
+interface SlotAvailability {
+  available: boolean,
+  offset: number,
+}
+
+const isSlotAvailable = ({
+  slotStartTime,
+  slotEndTime,
+  busyTimes
+}): SlotAvailability => {
+  let slotAvailability: SlotAvailability = {
+    available: true,
+    offset: 0,
+  }
+
+  for (let i = 0; i < busyTimes.length; i += 1) {
+    const busyTimeStart = dayjs(busyTimes[i].start);
+    const busyTimeEnd = dayjs(busyTimes[i].end);
+
+    // Check if start times are the same
+    if (slotStartTime.format('HH:mm') == busyTimeStart.format('HH:mm')) {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689309)
```suggestion
    if (slotStartTime.format("HH:mm") == busyTimeStart.format('HH:mm')) {
```